README.md

```diff
@@ -3,19 +3,25 @@
 
 ## Open Event Organizer App
 
-[![Build Status](https://img.shields.io/travis/fossasia/open-event-orga-app/master.svg?style=flat-square)](https://travis-ci.org/fossasia/open-event-orga-app)
-[![Codacy Grade](https://img.shields.io/codacy/grade/e27821fb6289410b8f58338c7e0bc686.svg?style=flat-square)](https://www.codacy.com/app/mb/open-event-orga-app)
-[![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg?style=flat-square)](https://gitter.im/fossasia/open-event-orga-app)
+[![Build Status](https://img.shields.io/travis/fossasia/open-event-orga-app/master.svg)](https://travis-ci.org/fossasia/open-event-orga-app)
+[![Codacy Grade](https://img.shields.io/codacy/grade/9a25fb73a9054144b384b5c6d62243f4.svg)](https://www.codacy.com/app/mb/open-event-orga-app)
+[![Codecov Coverage](https://img.shields.io/codecov/c/github/fossasia/open-event-orga-app.svg)](https://codecov.io/gh/fossasia/open-event-orga-app)
+[![Appetize Preview](https://img.shields.io/badge/Preview-appetize.io-673AB7.svg)](https://appetize.io/app/t8hace1jdrybzvwwrfw5p1ayuw)
+[![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg)](https://gitter.im/fossasia/open-event-orga-app)
+> Event management app for organizers using Open Event Platform
+
+![Icon](https://user-images.githubusercontent.com/3874064/29111547-5814a318-7d08-11e7-8c86-0a10a640825e.png)
```

Comment:
>It's standard 512 px ic_web for Android Projects Also, even screenshots will be there soon

yeah. screenshots will be good. but a logo in this size in readme doesn't make any sense. you can display logo after the title inline instead.